ReactAndroid/src/main/java/com/facebook/react/ReactInstanceManagerImpl.java

```diff
@@ -466,6 +466,22 @@ private void invokeDefaultOnBackPressed() {
     }
   }
 
+  @Override
+  public void onNewIntent(Intent intent) {
+    if (mCurrentReactContext == null) {
+      // Swallow the new intent?
+      FLog.w(ReactConstants.TAG, "Instance detached from instance manager");
+    } else {
+      String action = intent.getAction();
+
+      if (Intent.ACTION_VIEW.equals(action)) {
+        DeviceEventManagerModule deviceEventManagerModule =
```

Comment:
Modified. Pardon the sloppiness. Got in a hurry since need to check in on our Android release. If at any point it's easier to put this on a back burner or just make it right without trying to steer me in the right direction, don't hesitate 👍